harvester/cli.py

```diff
@@ -94,45 +94,45 @@ def parse_url_content(wacz_input_file: str, url: str) -> None:
     help="Filepath to WACZ archive from crawl",
 )
 @click.option(
-    "--metadata-output-file",
+    "--records-output-file",
     required=True,
-    help="Filepath to write metadata records to. Can be a local filepath or an S3 URI, "
+    help="Filepath to write records to. Can be a local filepath or an S3 URI, "
     "e.g. s3://bucketname/filename.jsonl.  Supported file type extensions: "
     "[jsonl, xml,tsv,csv].",
 )
 @click.option(
-    "--include-fulltext",
-    is_flag=True,
-    help="Set to include parsed fulltext from website in generated structured metadata.",
+    "--sitemap-urls-file",
+    required=False,
+    help="Filepath of URLs discovered from this crawl.",
 )
 @click.option(
-    "--extract-fulltext-keywords",
-    is_flag=True,
-    help="Set to use YAKE to extract keywords from fulltext.",
+    "--previous-sitemap-urls-file",
+    required=False,
+    help="Filepath of URLs discovered from previous crawl.",
```

Comment:
Hmm, I'm not sure I understand the purpose of `previous-sitemap-urls-file` and how it relates to `--sitemap-urls-file`. 🤔 Can you give an example?